frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -97,31 +149,33 @@ function VariableItem({
 					}}
 				/>
 			) : (
-				<Select
-					value={variableData.allSelected ? 'ALL' : variableData.selectedValue}
-					onChange={handleChange}
-					bordered={false}
-					placeholder="Select value"
-					mode={
-						(variableData.multiSelect && !variableData.allSelected
-							? 'multiple'
-							: null) as never
-					}
-					dropdownMatchSelectWidth={false}
-					style={{
-						minWidth: 120,
-						fontSize: '0.8rem',
-					}}
-					loading={isLoading}
-					showArrow
-				>
-					{variableData.multiSelect && variableData.showALLOption && (
-						<Option value={ALL_SELECT_VALUE}>ALL</Option>
-					)}
-					{map(optionsData, (option) => {
-						return <Option value={option}>{(option as string).toString()}</Option>;
-					})}
-				</Select>
+				!errorMessage && (
+					<Select
+						value={variableData.allSelected ? 'ALL' : variableData.selectedValue}
+						onChange={handleChange}
+						bordered={false}
+						placeholder="Select value"
+						mode={
+							(variableData.multiSelect && !variableData.allSelected
+								? 'multiple'
+								: null) as never
+						}
+						dropdownMatchSelectWidth={false}
+						style={{
```

Comment:
this will create a new object over render can you please put this over config and import from there since this is a static object only